website/project/views/node.py

```diff
@@ -38,7 +38,7 @@
 from website.profile import utils
 from website.project import new_folder
 from website.util.sanitize import strip_html
-
+from website.spam_admin.utils import _project_is_spam
```

Comment:
Not sure an '_' should be the first character on something that needs to be imported.